src/vorta/tray_menu.py

```diff
@@ -9,6 +10,11 @@ class TrayMenu(QSystemTrayIcon):
     def __init__(self, parent: QApplication = None):
         QSystemTrayIcon.__init__(self, parent)
         self.app = parent
+
+        self.forceWhite = False
+        if ("GNOME" in str(os.environ.get('XDG_CURRENT_DESKTOP')).upper()) or (platform.system() == "Darwin"):
+            self.force_white = True
```

Comment:
I suggest moving this into `set_tray_icon` or a new function to determine the colour dynamically.

Reply:
This only checks if the OS is Gnome based or MacOS and forces the ui to always use white tray icons. I think I'll be moving this to the set_tray_icon itself because some tests on linux gave attribute not found error due to not being able to find force_white attribute. I made this separate so that I could use it along with the uses_dark_mode method to determine the tray color but as the uses_dark_mode didn't work consistently I skipped it for now and only used force_white to determine tray icon color.

Reply:
I wrote a code using code from `uses_dark_mode` which dynamically changed the tray icon colour even while running, it detected the theme change in the os, but when implemented inside the `tray_menu.py` it didn't seem to work like intended. It didn't detect theme changes and when forced to check the theme it always gave `False`.